content/en/notification-manager/notification-manager.md

```diff
@@ -0,0 +1,404 @@
+---
+title: "Notification Manager"
+keywords: "kubernetes, kubesphere, notification manager, email, wechat, slack"
+description: "Multi-tenant notification manager on KubeSphere"
+---
+
+Notification Manager manages notifications in kubesphere. It receives alerts or notifications from different senders and then send notifications to various tenant receivers based on alerts/notifications' tenant label like "namespace". 
+
+Supported senders includes:
+- Prometheus Alertmanager
+- Custom sender (Coming soon)
+
+Supported receivers includes:
+- Email
+- [Wechat Work](https://work.weixin.qq.com/)
+- Slack 
+- Webhook (Coming soon)

```

Comment:
@wanjunlei  Will webhook be supported in v3.0?

Reply:
no